source/sdk/Numbers.ooc

```diff
@@ -95,17 +88,18 @@ Double: cover from double extends LDouble {
 	}
 }
 
+Double: cover from double extends LDouble {
+	isNumber ::= this == this
+	toString: func -> String {
+		"%.2f" formatDouble(this)
```

Comment:
What a confusing diff. Anyway, `"%.2f" formatDouble(this)` is now `toString` of both `LDouble` and `Double`, because of a bug in rock. https://github.com/fasterthanlime/rock/issues/445